uefi-test-runner/src/boot/memory.rs

```diff
@@ -68,6 +68,8 @@ fn memory_map(bt: &BootServices) {
     let first_desc = desc_iter.next().unwrap();
 
     let phys_start = first_desc.phys_start;
+    let page_count = first_desc.page_count;
 
     assert_eq!(phys_start, 0, "Memory does not start at address 0");
+    assert!(page_count != 0, "Memory map entry has zero size");
```

Comment:
Just another sanity check. The UEFI spec mandates this, and it can be used to check that we did not get a memory block full of zeroes...